workshops/migrations/0039_add_permission_groups.py

```diff
@@ -15,7 +15,7 @@ def add_permission_groups(apps, schema_editor):
     # create 'administrators' group with all permissions for CRUD
     auth_ct = ContentType.objects.get_for_models(Permission, Group)
     workshops_ct = ContentType.objects.get_for_models(
-        Airport, Award, Badge, Event, Host, KnowledgeDomain, Lesson, Person,
+        Airport, Award, Badge, Event, Organization, KnowledgeDomain, Lesson, Person,
```

Comment:
Hmm this migration is done before Organization model exists, shouldn't we use `Organization = apps.get_model('workshops', 'Organization')` or `Host = apps.get_model('workshops', 'Host')`?